src/sentry/conf/server.py

```diff
@@ -933,6 +935,12 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
         # TODO: Increase expiry time to x4 once we change this to run weekly
         "options": {"expires": 60 * 60 * 3},
     },
+    "schedule_auto_transition": {
+        "task": "sentry.tasks.schedule_auto_transition",
+        # Run every 15 minutes
```

Comment:
Running every 15 minutes was arbitrarily chosen based on other task schedules. Not sure if we need to tune this or not.

Reply:
Thanks for calling this out - let's start with a daily schedule and see how it scales as we go through internal/EA releases